### PR TITLE
Fix gpu sample bug and add same notebook sample

### PR DESCRIPTION
**Pull request by hongye-sun**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/605)

### CODE REVIEW

File: samples/kubeflow-tf/kubeflow-training-classification.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def kubeflow_training(output, project,
   # TODO: use the argo job name as the workflow
   workflow = '{{workflow.name}}'
   # set the flag to use GPU trainer
-  use_gpu = True
+  use_gpu = False
```
Comment (gaoning777) [inline samples/kubeflow-tf/kubeflow-training-classification.py]: Why change the use_gpu to false?

Comment (hongye-sun) [reply samples/kubeflow-tf/kubeflow-training-classification.py]: By default, the gpu is not enabled in kubeflow's deployment. I am going to add a doc about how to enable gpu in new/existing cluster and will put the doc link in comment here.

Comment (gaoning777) [reply samples/kubeflow-tf/kubeflow-training-classification.py]: SG

**Comment by hongye-sun** [comment]
/assign @Ark-kun @qimingj 
Need another lgtm label. Thanks.

**Comment by gaoning777** [comment]
Discussed offline that we need to add sample tests to cover this GPU testing

**Comment by hongye-sun** [comment]
The ml-pipeline-test doesn't have gpu quota. Requesting quota will take several days. I will revert my testing changes and continue without gpu disabled in sample in this PR.

**Pull request closed without merging**